lib/src/states/camera_picker_viewer_state.dart

```diff
@@ -141,12 +140,11 @@ class CameraPickerViewerState extends State<CameraPickerViewer> {
         );
       } catch (e, s) {
         handleErrorWithHandler(e, s, onError);
+      } finally {
+        safeSetState(() {
+          isSavingEntity = false;
+        });
       }
-      isSavingEntity = false;
-      if (mounted) {
-        setState(() {});
-      }
-      return;
```

Comment:
Gotcha! This is a very bad change... The "return" is omitted @AlexV525 . So the issue about duplicate saving entity has happened again.